Unable to build playroom without a favicon

I'm unable to run `playroom build` after upgrading to `0.34.1`. I'm getting the following error:

```
Error: HtmlWebpackPlugin: could not load file /path/to/repo/images/favicon.png
- index.js:494      [repo]/[html-webpack-plugin]/index.js:494:35     error Command failed with exit code 1.
```

This seems to have been caused by PR #309

If I create a `images/favicon.png` file everything works correctly, but this seems unexpected for a patch release.

--- a/.changeset/six-spoons-change.md
+++ b/.changeset/six-spoons-change.md
@@ -1,0 +1,5 @@
+---
+'playroom': patch
+---
+
+Fix `playroom build` by making favicon path relative to webpack config

--- a/lib/makeWebpackConfig.js
+++ b/lib/makeWebpackConfig.js
@@ -167,7 +167,7 @@ module.exports = async (playroomConfig, options) => {
         chunksSortMode: 'none',
         chunks: ['index'],
         filename: 'index.html',
-        favicon: 'images/favicon.png',
+        favicon: path.join(__dirname, '../images/favicon.png'),
         base: playroomConfig.baseUrl,
       }),
       new HtmlWebpackPlugin({

